docs/managed-datahub/observe/freshness-assertions.md

```diff
@@ -145,6 +145,8 @@ Freshness Assertions also have an off switch: they can be started or stopped at
 
 Once these are in place, you're ready to create your Freshness Assertions!
 
+You can also **Bulk Create** freshness assertions via the [Data Health Page](https://docs.datahub.com/docs/managed-datahub/observe/data-health-dashboard#bulk-create-smart-assertions)
```

Comment:
Freshness Assertions.

Also do we need bold here? We have different bolding on the volume doc